front/src/App.js

```diff
@@ -51,7 +52,7 @@ function App() {
   const handleSendMessage = event => {
     event.preventDefault()
 
-    socket.emit('chat message', event.target.input.value)
+    socket.emit('chat message', playerID, event.target.input.value) //playerID is taken from the state
```

Comment:
Again, I think it's best if we keep the `chat message` client-to-server messages as simply containing the message contents. The server-to-client `chat messages` will then have a `playerID` field with the value of `socket.id`.